creator-node/src/routes/files.js

```diff
@@ -229,7 +229,14 @@ const getCID = async (req, res) => {
       decisionTree.push({
         stage: `CID_CONFIRMED_FILE`
       })
-      fileFoundOnFS = true
+
```

Comment:
thoughts on moving this inside the streamFromFileSystem function? like here maybe https://github.com/AudiusProject/audius-protocol/blob/c81c271ab5babba10d95f1d56ebf839d3460b2b7/creator-node/src/routes/files.js#L75

Reply:
+1

Reply:
also we should follow the example from the card where we check that the CID is not the empty file
`if (fileIsEmpty && multihash !== EMPTY_FILE_CID)`
https://github.com/AudiusProject/audius-protocol/blob/master/creator-node/src/fileManager.js#L341

Reply:
i think i see the intent with separate logs for current and legacy storage path, we can instead just log `path` from streamFromFilesystem` which would tell us if legacy or new?

Reply:
no, I don't think we should move this functionality into `streamFromFileSystem`. this fn has one purpose, which is streaming from the fs and not removing from fs. if the file is actually empty, it should stream it as it does now. we should be adding this functionality in this block of code here bc this is where we check the file itself and handle each scenario as necessary

Reply:
can add the check for the empty cid

Reply:
yeah thats reasonable, sg

Reply:
the logs here were to highlight that the cid was removed from fs. my thinking was that this log will give us good insight on how often a file that shouldn't have been negative was removed from the fs

Reply:
agreed! log is useful here